### PR TITLE
[Compute] `az vmss update`: Add an option `Standard` to `--security-type` parameter

### PR DESCRIPTION
**Pull request by yanzhudd**
**Related command**

`az vmss update`

**Description**

Close: #28991

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Compute

File: src/azure-cli/azure/cli/command_modules/vm/_params.py [hunk 1]
```diff
@@ -828,6 +828,7 @@ def load_arguments(self, _):
         c.argument('enable_vtpm', enable_vtpm_type)
         c.argument('custom_data', help='Custom init script file or text (cloud-init, cloud-config, etc..)', completer=FilesCompleter(), type=file_type)
         c.argument('security_type', arg_type=get_enum_type(["TrustedLaunch"]), min_api='2020-06-01', help='Specify the security type of the virtual machine scale set.')
+        c.argument('security_type', arg_type=get_enum_type(["TrustedLaunch", "Standard"]), min_api='2023-07-01', help='Specify the security type of the virtual machine scale set.')
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/_params.py:831]: Just out of curiosity, is `Standard` only supported after api version `2023-07-01`?

Comment (yanzhudd) [reply src/azure-cli/azure/cli/command_modules/vm/_params.py:831]: correct, it is mentioned in the [feature request](https://github.com/Azure/azure-cli/issues/28991#issue-2305127576):
image

Comment (AjKundnani) [reply src/azure-cli/azure/cli/command_modules/vm/_params.py:831]: > This looks a bit strange
> 
> > Just out of curiosity, is Standard only supported after api version 2023-07-01?
> 
> @AjKundnani Could you please confirm this question?

@zhoxing-ms - Thanks for catching this, min API version for roll-back support is `2020-06-01`. Will update the request as well.
cc @yanzhudd

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**